### ISSUE TITLE
how to read SUNBURST chart

### ISSUE THREAD

**Issue by 304NotModified** [codecov/support#174]
Really cool the SUNBURST, but how do I read it? Can you point me to some docs?

![image](https://cloud.githubusercontent.com/assets/5808377/14939265/c9680e16-0f3f-11e6-9aee-0653ad3324f9.png)

**Comment by stevepeak** [comment]
Hey :) sorry, I don't have any docs on it yet. But basically the outermost blocks are files and as you move inward it's the folders. The center is the entire project. 

In your dashboard you can interact with the sunburst and it will update the path network below the sunburst. Which can help understand the graph.

Thanks!

**Comment by 304NotModified** [comment]
Thanks. I think tooltip on the parts would be nice.

**Comment by stevepeak** [comment]
Agreed :) I planned on adding one, but didn't make the deploy deadline. Thanks!

**Comment by Flamefire** [comment]
Yes we really need tootips on the blocks as otherwise it results in heavy clicking around to find useful folders ;)

**Comment by stevepeak** [comment]
@Flamefire does the breadcrumbs at the bottom help?
<img width="324" alt="screen shot 2016-05-06 at 2 35 51 pm" src="https://cloud.githubusercontent.com/assets/2041757/15082823/dd70a772-1397-11e6-9657-96ac20e14e1a.png">

**Comment by Flamefire** [comment]
Yes it does help. But you see it only AFTER clicking on a bar. E.g. there might be some folders, that you know you don't care about ATM but you still need to click them to see which ones they are. Showing the folder/file while hovering over the bar would be much better. Otherwise I doubt I'd use the chart at all.

**Comment by sfdc-bruce** [comment]
This really needs tooltips to be useful.  And a way to return to the top node with one click.

**Comment by stevepeak** [comment]
@sfdc-bruce Tooltips would be great! I'll look into that. You can return to the parent node by clicking the inner-most circle.
